docker-compose.yml

```diff
@@ -40,14 +41,16 @@ services:
       PS_FOLDER_ADMIN: ${PS_FOLDER_ADMIN:-admin-dev}
       PS_COUNTRY: ${PS_COUNTRY:-fr}
       PS_LANGUAGE: ${PS_LANGUAGE:-en}
-      PS_DEV_MODE: ${PS_DEV_MODE:-1}
+      PS_DEV_MODE: ${PS_DEV_MODE:-0}
```

Comment:
```suggestion
      PS_DEV_MODE: ${PS_DEV_MODE:-1}
```

Reply:
Yes, the Docker compose is used for the Nightly and must be in production mode.

Reply:
The docker-compose is a base for developers to work on the project, if you need the production mode you should set the PS_DEV_MODE env var to 0 wherever you need it, but not set it as the default in the base docker-compose, imho.

Reply:
The docker-compose is a base for the nightly since years. If there are changes for that, QA Automation team must be made aware of that.

Reply:
I think @0x346e3730 would say that developers can use this docker-compose to make PS run on their develop environment, and by default the dev mode will be disabled with your changes

Reply:
I fix it...